### DISCUSSION TITLE
How to disable the captive portal

### DISCUSSION THREAD

**Discussion by ramanraja** [arendst/Tasmota#14742] [Q&A] [answered]
I have an automated script to configure the WiFi credentials on my customers' Tasmota devices.
But as soon as I connect to the wireless AP of ESP 8266, the browser automatically opens the Wifi Manager config page. I understand this is because of the captive portal mechanism.
In the original WiFi Manager example code, I see this line:
`wm.setCaptivePortalEnable(false); // disable captive portal redirection`
I want to get the same effect within Tasmota. 
Please let me know if there is any preprocessor directive for this, or which part of the source code I need to edit.

**Comment by sfromis** [comment, marked as answer]
`#define NO_CAPTIVE_PORTAL` will change what code is generated, to remove captive portal code. Not tested it myself, I don't know details.
